### DISCUSSION TITLE
Can I override the git path being used by goreleaser?

### DISCUSSION THREAD

**Discussion by DaChartreux** [goreleaser/goreleaser#4350] [Q&A] [answered]
I have a deployment pipeline for my services, where we use git version 2.6.1. Now this doesn't support --format option for `git tag -l` (which I assume cannot be skipped). So I decided to install an updated git version to some other directory in such a way that it doesn't conflict with my existing pipeline. But now I don't know how to pass this updated git path to goreleaser.

Can someone help me out?

**Comment by caarlos0** [comment, marked as answer]
not directly, but i'll always use the first one it can find in the `PATH`, so, if you adjust it accordingly it should work:

```sh
PATH=/your/newer/git/bin/:$PATH goreleaser release ...
```
